src/blocks/recommendations/editor.scss

```diff
@@ -1,3 +1,9 @@
+// Fix wrong element height in FSE.
 .parsely-recommendations-card > div {
 	height: unset;
 }
+
+// Disable clicking on links.
+.parsely-recommendations-link {
+	pointer-events: none;
+}
```

Comment:
If we're going with the solution of having `#` as a link, we no longer need the CSS changes right?

Reply:
I thought about this, but I decided I preferred to leave it there, as clicking on `#` could still move the viewport which would be an annoyance. So I do think that keeping it makes sense.